docs/discord-social-sdk/development-guides/setting-rich-presence.mdx

```diff
@@ -247,6 +247,46 @@ client->UpdateRichPresence(
 ```
 ---
 
+## Setting Field URLs
```

Comment:
I think both of these sections would be better just below `## Setting Assets` since it flows naturally from there.

I am wondering though if that section should be `## Setting Predefined Assets` and this should be something like `## Setting Rich Presence Clickable Links` or something like that - that describes what you are doing.

What do you think of that?

Down here we're in `## Rich Presence Without Authentication` which feels like the wrong section.

Reply:
im good with moving them! i think that makes sense

i'm like 70% sure that you can set more than just predefined assets so i dont think renaming that section is necessary. as for renaming the field URLs section, i think it's redundant to say Rich Presence when we're on the rich presence page but i'm open to maybe doing "Setting Clickable Links" if that feels more intuitive to you than "Setting Field URLs"

so in the meantime ive moved the section, but left the headers the same. lmk your thoughts on if you still think the header could be reworded